pkg/version/version.go

```diff
@@ -40,3 +40,7 @@ func Get() apimachineryversion.Info {
 		Platform:     fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH),
 	}
 }
+
+func GetLatestVersion() string {
+	return "1.4.0"
```

Comment:
You can get from https://kubeedge.io/latestversion

Reply:
This can work with offline case, and update all version info in release process is a common way.

Reply:
We have a default version in `keadm/cmd/keadm/app/cmd/common/constant.go`

Reply:
Let's use it :)